tests/deploys_test.go

```diff
@@ -0,0 +1,61 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/neotest"
+	"github.com/nspcc-dev/neo-go/pkg/neotest/chain"
+	"github.com/nspcc-dev/neofs-contract/contracts"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDeploys(t *testing.T) {
+	bc, acc := chain.NewSingle(t)
+	e := neotest.NewExecutor(t, bc, acc, acc)
+
+	fsContracts, err := contracts.GetFS()
+	require.NoError(t, err)
+
+	for _, fsContract := range fsContracts {
+		if fsContract.Manifest.Name == "NameService" {
+			nnsContract := neotest.Contract{
+				Hash:     state.CreateContractHash(e.CommitteeHash, fsContract.NEF.Checksum, fsContract.Manifest.Name),
+				NEF:      &fsContract.NEF,
+				Manifest: &fsContract.Manifest,
+			}
+
+			e.DeployContract(t, &nnsContract, []any{[]any{[]any{"neofs", "ops@nspcc.io"}}})
+		}
+
+		if fsContract.Manifest.Name == "NeoFS Netmap" {
+			netMapContract := neotest.Contract{
+				Hash:     state.CreateContractHash(e.CommitteeHash, fsContract.NEF.Checksum, fsContract.Manifest.Name),
+				NEF:      &fsContract.NEF,
+				Manifest: &fsContract.Manifest,
+			}
+
+			netMapContractDeployData := []any{
+				false, // notaryDisabled is false
+				nil,
+				nil,
+				nil,
+				[]any{},
+			}
+
+			e.DeployContract(t, &netMapContract, netMapContractDeployData)
+
+			params := NewRegisterContractInNNSParams("netmap", netMapContract.Hash)
```

Comment:
following steps should be done outside the loop

Reply:
Reworked the code a bit